sycl/source/detail/graph_impl.cpp

```diff
@@ -667,6 +677,23 @@ exec_graph_impl::enqueueNodeDirect(sycl::context Ctx,
   }
   ur_exp_command_buffer_sync_point_t NewSyncPoint;
   ur_exp_command_buffer_command_handle_t NewCommand = 0;
+
+#ifdef XPTI_ENABLE_INSTRUMENTATION
```

Comment:
Can we have a `sycl/unittests/xpti_trace` test(s) using graphs if we're expecting this to work and stay working.

Reply:
Sure, added in new commit.
Thanks!